samples/performance/command_buffer_usage/command_buffer_usage.cpp

```diff
@@ -149,7 +149,7 @@ void CommandBufferUsage::draw_gui()
 	    /* body = */ [&]() {
 		    // Secondary command buffer count
 		    ImGui::PushItemWidth(ImGui::GetWindowWidth() * 0.55f);
-		    ImGui::SliderInt("", &gui_secondary_cmd_buf_count, 0, max_secondary_command_buffer_count, "Secondary CmdBuffs: %d");
+		    ImGui::SliderInt("Secondary CmdBuffs: %d", &gui_secondary_cmd_buf_count, 0, max_secondary_command_buffer_count);
```

Comment:
Something isn't quite right here.
I see "Secondary CmdBuffs: %d Draws/buf: 0.0" on my display.

Reply:
Will look into it. Possibly don't need the %d.

I moved this as it crashes on an imgui assertion

Reply:
<img width="943" alt="Screenshot 2024-05-29 at 23 02 07" src="https://github.com/KhronosGroup/Vulkan-Samples/assets/6977035/f8169a1b-474a-4232-b1ff-651aa21bdd42">

Yup was the %d